### ISSUE TITLE
Configuration Options - PlaylistPath

### ISSUE THREAD

**Issue by spiral6** [navidrome/website#62]
Need to add playlist path config option

**Comment by deluan** [comment]
Yes, it will be added once the feature is released, probably this week. It is not documented yet because the documentation only reflects what is in the latest release, not development builds. Thanks for the reminder anyway :)
